speculos.py

```diff
@@ -28,6 +28,8 @@
 from mcu.finger_tcp import FakeFinger
 from mcu.vnc import VNC
 
+LOG = logging.getLogger("speculos")
```

Comment:
Please keep `logger` in lowercase

Reply:
Fixed in 5f43cc1225b3dd34cc47aa1e0013b8b6badaa938.